### PR TITLE
Tmz 3.2.0 enhancement dtmf issue 1234

### PR DESCRIPTION
**Pull request by t-m-z**
play dtmf tones

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -23,6 +23,10 @@ dependencies:
   file_picker_writable: ^2.0.3
   flutter:
     sdk: flutter
+  flutter_dtmf: ^3.0.2
+  # flutter_dtmf:
```
Comment (Mike-3) [inline pubspec.yaml:27]: remove comments

File: lib/tools/science_and_technology/dtmf/logic/dtmf.dart [hunk 2]
```diff
@@ -1,3 +1,4 @@
+//import 'package:audioplayers/audioplayers.dart';
```
Comment (Mike-3) [inline lib/tools/science_and_technology/dtmf/logic/dtmf.dart:1]: remove comments

**Review by Mike-3 — CHANGES_REQUESTED** [review]

File: lib/tools/science_and_technology/dtmf/logic/dtmf.dart (hunk 2 above)
Comment (t-m-z) [reply lib/tools/science_and_technology/dtmf/logic/dtmf.dart]: done

File: pubspec.yaml (hunk 1 above)
Comment (t-m-z) [reply pubspec.yaml]: done

**Pull request merged**